tox.ini

```diff
@@ -19,6 +19,7 @@ deps =
     py39: molecule-docker
     # molecule dep that is incompatible with python 2 as of 1.13.0 & 1.13.1
     sh < 1.13 ; python_version < "3"
+    ruamel.yaml < 0.17 ; python_version < "3"
```

Comment:
Why not "py27: ..." ?

Reply:
I just followed the way `sh` was being handled, but I think `py27: ...` would work also